_contentTemplates/common/get-started.md

```diff
@@ -130,6 +130,8 @@ To setup a local NuGet package source, so you can install the Telerik components
 
 
 #root-component-telerik-layout
+To enable the use of detached popups (for example, dropdown lists, menus, grid filters, etc.), you must add a component at the root level of the DOM:
```

Comment:
"detached" may raise questions such as "which popups are not detached" or "how to use non-detached".

"a component" may rase a question "what component" if the reader doesn't continue reading immediately.

```suggestion
To use popups (for example, dropdowns, menus, windows, grid filters, etc.), you must add a `TelerikRootComponent` component at the root level of the app:
```